solr8.11.2_files/solrCloud_embedded_zooKeeper/init_files/security.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
We should document these are example/default passwords not to be used in production.

Reply:
I have removed this file. For the next sprint, I will create a task to clean up and generate better documentation of this repository. I have created different ways of setting Solr for testing but I should update the documentation to include a clear explanation of how we are setting up Solr Cloud for local development with docker and also in Kubernetes (development and production)